test/f32-vscaleextexp.cc

```diff
@@ -4,7 +4,7 @@
 // LICENSE file in the root directory of this source tree.
 //
 // Auto-generated file. Do not edit!
```

Comment:
When you remove the generator script, also remove this comment, because this file will not be generated code any more.